src/Deployment/Deployer.php

```diff
@@ -260,6 +263,21 @@ private function uploadPaths(array $paths)
 			$this->writeProgress($num + 1, count($paths), $path, NULL, 'green');
 		}
 
+		if ($this->runAfterUpload) {
+			$this->logger->log("\nAfter-upload-jobs:");
+			$this->runJobs($this->runAfterUpload);
+		}
+
+		$this->renameFiles($toRename);
+	}
+
+
+	/**
+	 * Renames uploaded files.
+	 * @return void
+	 */
+	private function renameFiles(array $toRename)
```

Comment:
Please use here $files and the same @param as is used in deletePaths. Name renameFiles is ok.